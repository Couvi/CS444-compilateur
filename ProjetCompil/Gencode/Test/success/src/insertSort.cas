program
	A : array [1..10] of integer;
	i, j, value : integer;
begin

A[1] := 24;
A[2] := 34;
A[3] := 4;
A[4] := 54;
A[5] := 24;
A[6] := 1;
A[7] := 3;
A[8] := 1;
A[9] := 67;
A[10] := 39;

for i := 2 to 10 do
	value := A[i];
	j := i-1;
	wHILe (j > 0) and (A[j] > value) do
		A[j+1] := A[j];
		j := j-1;
	end;
	A[j+1] := value;
end;
for i := 1 to 10 do
	write(A[i],", ");
end;
end.
	
