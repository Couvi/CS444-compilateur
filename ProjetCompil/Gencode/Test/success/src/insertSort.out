1, 1, 3, 4, 24, 24, 34, 39, 54, 67, 
